lib/util.js

```diff
@@ -3,26 +3,22 @@
  */
 
 /**
- * A utility function to create a verification suite.
+ * A utility function to create a multibase-multikey deserializer function.
  *
  * @param {object} options - Options hashmap.
- * @param {Function} options.generate - A method that generates a KeyPair with
- *   an optional deterministic seed.
- * @param {Function} options.from - A method that creates a verification KeyPair
+ * @param {Function} options.from - A method that creates a KeyPair
```

Comment:
I don't think we need this other `from` method any more with the simplification below.